seacrowd/sea_datasets/muse/muse.py

```diff
@@ -0,0 +1,161 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Contains 110 large-scale ground-truth bilingual dictionaries created and released by Meta using an internal translation tool.
+The dictionaries account for polysemy. The data comprises of a train and test split of 5000 and 1500 unique source words, as well as a larger set of up to 100k pairs.
+It comprises of Europeans languages in every direction, and SEA languages to and from English.
+"""
+
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_CITATION = """\
+@inproceedings{lample2018word,
+  title={Word translation without parallel data},
+  author={Lample, Guillaume and Conneau, Alexis and Ranzato, Marc'Aurelio and Denoyer, Ludovic and J{\'e}gou, Herv{\'e}},
+  booktitle={International Conference on Learning Representations},
+  year={2018}}
+}
+"""
+
+_DATASETNAME = "muse"
+
+_DESCRIPTION = """\
+Contains 110 large-scale ground-truth bilingual dictionaries created and released by Meta using an internal translation tool.
+The dictionaries account for polysemy. The data comprises of a train and test split of 5000 and 1500 unique source words, as well as a larger set of up to 100k pairs.
+It comprises of Europeans languages in every direction, and SEA languages to and from English.
+"""
+
+_HOMEPAGE = "https://github.com/facebookresearch/MUSE#ground-truth-bilingual-dictionaries"
+
+_LANGUAGES = ["fil", "ind", "zlm", "tha", "vie"]
+
+_LICENSE = Licenses.CC_BY_NC_ND_4_0.value
+
+_LOCAL = False
+
+_TRAIN_URL_TEMPLATE = "https://dl.fbaipublicfiles.com/arrival/dictionaries/{src}-{tgt}.0-5000.txt"
+_TEST_URL_TEMPLATE = "https://dl.fbaipublicfiles.com/arrival/dictionaries/{src}-{tgt}.5000-6500.txt"
+
+_SUPPORTED_TASKS = [Tasks.MACHINE_TRANSLATION]
+
+_SOURCE_VERSION = "1.0.0"
+_SEACROWD_VERSION = "1.0.0"
+
+configs = {"tl": ["en"], "id": ["en"], "ms": ["en"], "th": ["en"], "vi": ["en"], "en": ["th", "vi", "tl", "ms", "id"]}
+
+
+class MUSEDataset(datasets.GeneratorBasedBuilder):
+    """Large-scale ground-truth bilingual dictionaries"""
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+
+    BUILDER_CONFIGS = (
+        [
+            SEACrowdConfig(
+                name=f"{_DATASETNAME}_source",
```

Comment:
No need to have a default `{_DATASETNAME}_source` subset.

Reply:
@holylovenia it seems that `python -m tests.test_seacrowd seacrowd/sea_datasets/<dataset_name>/<dataset_name>.py` expects our dataloader to have a default `muse_source` and `muse_seacrowd_t2t`? Should i just ignore the outputs of that test?